manifests/h/Hellofont/Hellofont/3.0.3/Hellofont.Hellofont.installer.yaml

```diff
@@ -0,0 +1,28 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=QUSU.7-2-3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Hellofont.Hellofont
+PackageVersion: 3.0.3
+MinimumOSVersion: 10.0.0.0
+InstallerType: nullsoft # electron-builder
+InstallerSwitches:
+  Upgrade: --updated
```

Comment:
Upgrade: --updated
Silent: /S

Reply:
The `InstallerType` has been set to nullsoft and the `/S` switch will be automatically appended by WinGet.